lib/datadog/core/crashtracking/tag_builder.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require_relative '../utils'
+require_relative '../environment/socket'
+require_relative '../environment/identity'
+require_relative '../environment/git'
+
+module Datadog
+  module Core
+    module Crashtracking
+      # This module builds a hash of tags
+      module TagBuilder
+        def self.call(settings)
+          hash = {
+            'host' => Environment::Socket.hostname,
+            'language' => Environment::Identity.lang,
```

Comment:
I don't think this is needed, as the crashtracker API already gets this as a direct argument (in the C code)

```suggestion
```